source/features/sort-conversations-by-update-time.tsx

```diff
@@ -4,6 +4,11 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import SearchQuery from '../github-helpers/search-query';
 
+function selectCurrentConversationFilter(): void {
+	const currentSearchURL = location.href.replace('/pulls?', '/issues?'); // Replacement needed to make up for the redirection of "Your pull requests" link
+	select(`#filters-select-menu a.SelectMenu-item[href="${currentSearchURL}"]`)?.setAttribute('aria-checked', 'true');
```

Comment:
Marks the correct current filter as selected, just like in native GitHub.

Reply:
I suppose this would work. My concern is that I don't see a `aria-checked = false` anywhere here, so potentially we're leaving pre-selected options unchanged.

Maybe it should look like:

```js
const currentItem = select(etc)

if (currentItem)
	select(aria-selected=true).setAttribute(false)
	currentItem.setAttribute(true)